docs/deploy/k8s/index.mdx

```diff
@@ -0,0 +1,21 @@
+# Kubernetes and Weaviate
+
+Weaviate leverages Kubernetes for scalable, resilient, and production-grade deployments. 
+
+## Deployment Options
+
+Weaviate can use Kubernetes in a variety of deployments:
+
+### Enterprise Cloud
+
+- Amazon EKS (Elastic Kubernetes Service)
+- Google Kubernetes Engine (GKE)
+- Azure Kubernetes Service (AKS)
+
+### Bring Your Own Cloud
```

Comment:
Do we offer all of these BYOC providers? I know that we are now working on an automated BYOC solution called mBYOC (managed BYOC), but it currently only supports AWS.